docs/coordinates/angles.rst

```diff
@@ -330,7 +330,7 @@ Comparing Spherical Point Generation Methods
         pts = func(size=128)
 
         xyz = pts.to_cartesian().xyz
-        ax.plot(*xyz, ls='none')
+        ax.scatter(*xyz)
```

Comment:
Oops! Thanks for catching this. I slightly prefer:
```suggestion
        ax.plot(*xyz, ls='none', marker='o')
```
for cases like this where all points have the same styling because `plot()` will be much faster for large numbers of points.

Reply:
The two options do not produce the same plot. `ax.scatter()` has the `depthshade` parameter, which can help with conveying depth, but `ax.plot()` does not seem to have such a capability.

Reply:
Interesting -- I didn't know about the `depthshade` option! OK in that case, fine to switch to scatter.